testing/steps/test_column_transformer.py

```diff
@@ -0,0 +1,389 @@
+import copy
+from typing import Any
+
+import numpy as np
+import pytest
+from neuraxle.base import BaseStep
+from neuraxle.steps.column_transformer import ColumnTransformer
+
+
+class MultiplyBy2(BaseStep):
```

Comment:
I would like that you use the multiply we already coded to avoid duplicated code.